desktop/src/gui/menu_bar.rs

```diff
@@ -64,7 +66,14 @@ impl MenuBar {
                 player.set_is_playing(!player.is_playing());
             }
         }
-        if egui_ctx.input_mut(|input| input.consume_shortcut(&Self::SHORTCUT_FULLSCREEN)) {
+        let mut fullscreen =
+            egui_ctx.input_mut(|input| input.consume_shortcut(&Self::SHORTCUT_FULLSCREEN));
+        if cfg!(windows) && !fullscreen {
```

Comment:
But it doesn't.

Reply:
Wait, I think I misunderstood something, sorry.

Reply:
The variable `fullscreen` tells whether the fullscreen shortcut was pressed, the code that toggles fullscreen is few lines below: https://github.com/ruffle-rs/ruffle/pull/18037/files#diff-f8a52a3fe6f612b8157bf85e7e2fb1cfaaaf771d54d44e6878a9349805dcb120R79

Reply:
And done, it's now `fullscreen_pressed`